### DISCUSSION TITLE
laravel save fresh

### DISCUSSION THREAD

**Discussion by iamamirsalehi** [laravel/framework#35577] [Show and tell]
Hi everyone, I hope you're doing great.
I'm working on a package that saves your table data after fresh your migrations (called laravel-save-fresh) and It's about to finish. I just wanna get your feedback and to know what's your opinion and suggestion about it. 
[https://github.com/iamamirsalehi/laravel-save-fresh](https://github.com/iamamirsalehi/laravel-save-fresh)

**Comment by kjostling** [comment]
link is broken

**Reply by myckhel** [reply]
https://github.com/iamamirsalehi/laravel-save-fresh

**Reply by driesvints** [reply]
I fixed the link in the main post.

**Reply by abbood** [reply]
link is still broken
